style.css

```diff
@@ -1,103 +1,98 @@
 }
 
-.nav {
-    display: flex;
-    align-items: center;
-    margin: 1em;
+header span hr {
+ border: 2px solid turquoise;
+ margin-top: 1px;
+ margin-bottom: 4px;
 }
 
-.nav ul {
-    display: flex;
-    margin-left: 3em;
-    gap: 0.4em;
+.wrapper {
+ display: flex;
+ justify-content: space-around;
+ margin-top: 20px;
 }
 
-.nav ul a {
-    padding: 5px;
-    border: 1px;
-    border-style: solid;
-    border-color: black;
-    border-radius: 10px;
-    text-decoration: none;
+.wrapper h2 {
+background-color: rgb(255, 255, 255, 0.8);
+padding: 10px 80px 10px 10px;
+margin-left: 10px;
 }
 
-.hero {
-    display: flex;
-    background-image: url("hero-image.jpg");
-    background-repeat: no-repeat;
-    background-size: cover;
-    width: 100vmax;
-    height: 65vmax;
-    margin-top: 2em;
-    justify-content: center;
-    
+.wrapper hr {
+ margin: 0px;
 }
 
-.hero h1 {
-    font-size: 7em;
-    font-weight: 700;
-    color: white;
-    height: auto;
-    width: 550px;
-    margin-top: 1em;
+.wrapper .hero {
+ height: 125px;
+ width: 300px;
+ display: flex;
+ align-items: center;
 }
 
-.blog {
-    display: flex;
-    flex-flow: column;
-    margin-left: 13em;
-    margin-right: 13em;
-    margin-top: 2em;
-    margin-bottom: 2em;
-    gap: 4em;
+.wrapper .text-content {
+ display: flex;
+ align-items: center;
+ padding: 5px 5px 5px 10px;
+ height: 125px;
+ border-top: 1px solid  black;
+ border-bottom: 1px solid  black;
 }
 
-.article {
-    border: 1px;
-    border-style: solid;
-    border-color: black;
-    }
+.text-content li:hover {
+ color: turquoise;
+}
+
+.wrapper ul ul {
+list-style-type: disc;
+}
+
+.wrapper .fruits {
+ display: flex;
+ flex-direction: column;
+ border: 1px solid black;
+ border-radius: 5px;
 
-.article h2 {
-    font-size: 20px;
-    font-weight: 700;
-    padding: 25px 7px;
 }
 
-#text {
-    padding: 0px 7px 15px;
+
+.wrapper .fruits .hero {
+ background-image: url(/fruits.png);
 }
 
-#date {
-    padding: 0px 7px 10px;
+.wrapper .veggies {
+ display: flex;
+ flex-direction: column;
+ border: 1px solid black;
+ border-radius: 5px;
 }
 
+.wrapper .veggies .hero {
+ background-image: url(/veggies.jpg);
+}
 
-footer {
-    font-size: 12px;
-    padding: 1em 0em;
-    margin-left: 0.5em;
+.wrapper .dairy {
+ display: flex;
+ flex-direction: column;
+ border: 1px solid black;
+ border-radius: 5px;
 }
 
-@media only screen and (max-width: 800px) {
-    .blog {
-        margin-left: 2em;
-        margin-right: 2em;
-      }
-      
-      .hero {
-          width: 100vmin;
-          height: 50vmax;
-      }
+.wrapper .dairy .hero {
+ background-image: url(/Dairy.jpg);
+}
 
-      .hero h1 {
-        font-size: 4em;
-      }
+label {
+ display: flex;
+ flex-direction: row-reverse;
+ align-items: center;
+ padding: 10px 10px 10px 0px;
+ background-color: turquoise;
+}
 
-      .nav ul {
-        margin-left: 1em;
-        gap: 0.4em;
-    }
+footer p {
+ font-size: 22px;
+ margin-left: 10px;
+ padding-top: 15px;
```

Comment:
ogólnie jeżeli chodzi o style
* zwracaj prosze na nazwy cssowe, niech są jasne semantycznie
* nie styluj prosze po tagach
* `.wrapper .veggies .hero` imo spokojnie mozna by było ominać klase `.wrapper` troche nie rozumie w ogóle idei tej klasy `.wrapper`, w sensie tego ze ja dodajesz do wieszkosci styli w selektorze jak ktoś by to zmienił na np. `.container` to Ci wsio poleci :)